README.md

```diff
@@ -96,4 +99,37 @@ To generate a new encrypted value or to replace one of the provided values, for
 $ ansible-vault encrypt_string --vault-password-file your_password_file --name 'trello_api_key'
 ```
 
-then enter the value you want to encrypt when prompted.
\ No newline at end of file
+then enter the value you want to encrypt when prompted.
+
+## Docker
+
+Run the project:
+```bash
+docker build --tag todo-app .
+docker run -dit \
+    --name todo-app \
+    -p 8000:8000 \
+    --env-file .env \
+    todo-app
+```
+
+Run the project with mounting:
+```bash
+docker build --target development --tag todo-app:dev .
+docker run -dit \
+    --name todo-app:dev \
+    -p 8000:8000 \
+    --env-file .env \
+    --mount type=bind,source="$(pwd)/todo_app",target=/app/todo_app,readonly \
```

Comment:
I don't think the mount is doing anything here because of Gunicorn, which doesn't hot reload. Check it out, e.g. load this up, then break your code some way (e.g. I just break the `create_app` function like adding a comment halfway through `def create#_app():`) and you'll see the app carries on happily anyway

Reply:
Hot reload here does actually work! Not sure why. For the prod one it doesn't since I've explicitly set `FLASK_DEBUG=true`